tile/codegen/aggrinit.cc

```diff
@@ -78,7 +80,7 @@ void AggregationBlockOutputInitializationPass::Apply(CompilerState* state) const
   for (const auto& toInit : this->state.blocksWithInits) {
     assert(toInit.blockToAddTo != nullptr);
     assert(toInit.refToInitialize != nullptr);
-    assert(toInit.initType == AggregationInitType::ADD || toInit.initType == AggregationInitType::MULL ||
+    assert(toInit.initType == AggregationInitType::ADD || toInit.initType == AggregationInitType::MUL ||
```

Comment:
I think this change needs to change the definition of the AggregationInitType::MUL and I seem to miss that change.

Reply:
Oh... that was the in assert and it is a typo. NM